### DISCUSSION TITLE
iconv-lite nearly doubles my page size, is this right?

### DISCUSSION THREAD

**Discussion by michaelketzer** [vercel/next.js#10888] [Help] [answered]
I have analyzed my pages and noticed that `iconv-lite` is actually the nearly a half of my pages bundle size.
Nextjs has enabled tree shaking as default, what I have gathered from several topics, but I have the feeling something is not right.

I setup an **empty** page. I have no imports in _document or _app, and this is enough for the following content:
<img width="1295" alt="Bildschirmfoto 2020-03-08 um 18 29 13" src="https://user-images.githubusercontent.com/46134133/76167942-c81d1c00-616a-11ea-8ace-ad00f489403c.png">

Is this the normal page size? Or did I miss anything out there? I have no `.babelrc` or anything else.
My source code from the analyzed image above is here:
https://gitlab.com/shokz.tv/web-v2/app/-/tree/73c159873e0f52163552ebe3502aca2a7f04d49e

The page is at `src/pages/event/[eventId].tsx`

My build output for this page:
<img width="582" alt="Bildschirmfoto 2020-03-08 um 18 27 43" src="https://user-images.githubusercontent.com/46134133/76167900-82f8ea00-616a-11ea-8b50-0e3988097788.png">

I have the feeling that something is not correct there, but it is just a guess.
When using gzip compression, the `iconv-lite` is actually more than the half of the my bundle:
<img width="1285" alt="Bildschirmfoto 2020-03-08 um 18 36 08" src="https://user-images.githubusercontent.com/46134133/76168023-b12af980-616b-11ea-80de-4c6801dcc7a5.png">

I've also looked thru the json files, they are mostly some chinese/japanese characters, I'm pretty sure I do not need them at all in my app or require any encoding for them.

**Comment by Timer** [comment, marked as answer]
It appears as-if you are analyzing the server-bundle instead of client bundle. Can you confirm you're inspecting files in the `.next/static/` directory?

**Reply by michaelketzer** [reply]
Oh that is the reason, i'm sorry for wasting the time on this issue then.
